### DISCUSSION TITLE
Replying to tweets is long-winded, convoluted and frustrating

### DISCUSSION THREAD

**Discussion by uniquePWD** [robertodoering/harpy#766] [General]
### Bug description 
Replying to tweets is a terrible user experience and can be improved a lot. 

---
### Expected behavior
When pressing the comment button, you should be given the opportunity to reply.  Instead you're presented with the existing replies. Meaning that replying is always two taps away. 

---
### Steps to reproduce
1. Go to your timeline
2. Click on the comments button 
3. See no reply box 
4. Attempt to reply to a reply
5. See no reply box 

### Proposed solution
Have access to the reply box page take precedence over displaying the thread. 

---
Android 13
Pixel 4XL

**Comment by robertodoering** [comment]
Currently you can reply to a Tweet either by tapping on the 3 dots on the top right of the Tweet card and selecting 'reply' or by tapping on the Tweet / comments button to open the Tweet details and then clicking on the reply button.

If you are _really_ into replying to Tweets and 2 taps is too much, fear not as with the planned Tweet card customization #304 you'll be able to add the reply button straight into the Tweet card.

**Reply by uniquePWD** [reply]
If you tap a tweet, you're taken to the subsequent comments, so why would you then add an additional button to do the same thing rather than have that button to reply. It's just an odd user experience.

**Reply by robertodoering** [reply]
That's what the 3-dot menu is for, which you can access by clicking on the 3 dots at the top right of a tweet.

**Reply by monee121** [reply]
I would prefer the 3 dot menu to have an option to view replies, and have the button as a 1 tap option to reply to a tweet. 

I'm used to having a quick option whether tap or swipe to reply to a tweet. Especially when I can't view more than a couple of replies in Harpy vs other 3rd party apps I can see several more.

**Reply by dnestico** [reply]
Nice! Can't wait for the tweet card customization.

**Comment by scottiefalkon** [comment]
I don't mind the current implementation at all. I usually like to see other
comments before I respond.

I would like to see harpy in the share sheet. For sharing articles and so
forth.

Gorgeous app. Appreciate all the hard work.

**Comment by scottiefalkon** [comment]
Perhaps a drop-down like on the retweet button. A tap gives a drop-down
that offers to view replies or reply.

**Reply by uniquePWD** [reply]
but the question stands, why add an additional button to do the same thing as pressing anywhere on a tweet will do? If you open a tweet, it shows all subsequent replies anyway.
